config/interceptors/core-interceptors.yaml

```diff
@@ -0,0 +1,70 @@
+# Copyright 2020 The Tekton Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: triggers.tekton.dev/v1alpha1
+kind: Interceptor
+metadata:
+  name: cel
+spec:
+  clientConfig:
+    url: "http://tekton-triggers-core-interceptors.tekton-pipelines.svc/cel"
```

Comment:
Letting the users set the path definitely seems useful, but it's unclear to me how users would know what Service URL this would be associated with, particularly if there are multiple interceptor services at play.

Few ideas:
1) Is the full URL something we can expose in the status and let the Interceptor controller handle filling this in for clients?
2) Should these CRDs be produced by Interceptor providers instead (e.g. on startup/control loop)? This has the added benefit of not requiring users to apply the interceptors themselves, sidestepping the apply ordering issues.

Reply:
end users do not have to know what the URL is. Only the interceptor provider. I implemented `url` first since that's the simplest but we will [support a ref to an object too ](https://github.com/tektoncd/community/blob/master/teps/0026-interceptor-plugins.md#installing-interceptors). 

> Is the full URL something we can expose in the status and let the Interceptor controller handle filling this in for clients?

Yes, was planning on adding this as part of the followup that adds service ref support. That being said, it might still make sense to have the same URL resolution happen at the EL level in case the EL receives an event before the Interceptor controller does a reconcile loop.

> Should these CRDs be produced by Interceptor providers instead (e.g. on startup/control loop)? This has the added benefit of not requiring users to apply the interceptors themselves, sidestepping the apply ordering issues.

Interesting idea. Maybe? I think it will solve the issue only for the core interceptors that are bundled with the Triggers installation. Otherwise, it will be up to the interceptor provider to ship both the deployment running the logic and the CRD to go with it. They could decide to ship a deployment that creates the CRD on startup (but then that deployment will need to have RBAC roles to create Interceptors)

Reply:
> but then that deployment will need to have RBAC roles to create Interceptors

mmmmm good point. Perhaps this is better then.